script/release_utils.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+function commit_list {
+  local tag=${1:?}
+  local repo_domain=${2:?}
+  local repo_name=${3:?}
+  git fetch --tags
+  local previous_tag=`curl -H "Authorization: token $ACCESS_TOKEN" -s https://api.github.com/repos/$repo_domain/$repo_name/tags | jq --raw-output '.[1].name'`
+  local commit_list=`git log $previous_tag..$tag --oneline`
```

Comment:
fyi this is the command I've been using to get the git log for the previous release notes: `git log --pretty=format:"- %s %H (%an)" v1.0.0-alpha.3..` as seen in https://github.com/kubeapps/kubeapps/releases

Reply:
👍 that format includes more info